radixdlt-java/src/main/java/com/radixdlt/client/application/translate/tokenclasses/CreateTokenToParticlesMapper.java

```diff
@@ -56,6 +56,12 @@
 				),
 				null
 		);
+
+		if (tokenCreation.getInitialSupply().isZero()) {
```

Comment:
Why is this required now and was not necessary before?

Reply:
Minting with initial supply never worked previously.  It should have for multi-issuance tokens.